### PR TITLE
feat: 🎸 1.17 app homepage - show avatar & copy address &  hide user's assets

### PR DESCRIPTION
**Pull request by wade-portkey**

### CODE REVIEW

File: packages/mobile-app-did/js/assets/theme/index.ts
Comment (Portkey-David) [inline packages/mobile-app-did/js/assets/theme/index.ts:1]: use new color key

File: packages/mobile-app-did/js/pages/DashBoard/CopyUserAddress/index.tsx
Comment (Portkey-David) [inline packages/mobile-app-did/js/pages/DashBoard/CopyUserAddress/index.tsx:1]: useCaAddressInfoList

Comment (Portkey-David) [inline packages/mobile-app-did/js/pages/DashBoard/CopyUserAddress/index.tsx:1]: key={index+""} ?  not necessary

File: packages/store/store-ca/wallet/slice.ts
Comment (Portkey-David) [inline packages/store/store-ca/wallet/slice.ts:1]: reset hideState  when quit wallet, need check this in useLogout hook.

File: packages/mobile-app-did/js/assets/theme/index.ts
Comment (wade-portkey) [reply packages/mobile-app-did/js/assets/theme/index.ts:1]: fixed

File: packages/mobile-app-did/js/pages/DashBoard/CopyUserAddress/index.tsx
Comment (wade-portkey) [reply packages/mobile-app-did/js/pages/DashBoard/CopyUserAddress/index.tsx:1]: fixed

Comment (wade-portkey) [reply packages/mobile-app-did/js/pages/DashBoard/CopyUserAddress/index.tsx:1]: fixed

File: packages/store/store-ca/wallet/slice.ts
Comment (wade-portkey) [reply packages/store/store-ca/wallet/slice.ts:1]: not need, the data will follow user

**Review by FrankFeng1116 — APPROVED** [review]

File: packages/mobile-app-did/js/components/TokenListItem/index.tsx [hunk 1]
```diff
@@ -58,10 +59,10 @@ const TokenListItem: React.FC<TokenListItemType> = props => {
         {!noBalanceShow && (
           <View style={itemStyle.balanceWrap}>
             <TextL style={itemStyle.token} numberOfLines={1} ellipsizeMode={'tail'}>
-              {formatTokenAmountShowWithDecimals(item.balance, item.decimals)}
+              {hideBalance ? '****' : formatTokenAmountShowWithDecimals(item.balance, item.decimals)}
```
[the remark this answers is not in the document]
Comment (wade-portkey) [reply packages/mobile-app-did/js/components/TokenListItem/index.tsx:62]: not magic number

File: packages/mobile-app-did/js/pages/DashBoard/index.tsx [hunk 2]
```diff
@@ -115,8 +166,29 @@ const DashBoard: React.FC<any> = ({ navigation }) => {
 };
 
 const styles = StyleSheet.create({
+  leftDomWrap: {
+    marginLeft: pTd(16),
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  accountName: {
+    color: defaultColors.font11,
```
[the remark this answers is not in the document]
Comment (wade-portkey) [reply packages/mobile-app-did/js/pages/DashBoard/index.tsx]: fixed

**Pull request merged**